management/prompt.go

```diff
@@ -0,0 +1,24 @@
+package management
+
+type PromptSettings struct {
+	// Which login experience to use. Can be `new` or `classic`.
+	UniversalLoginExperience string `json:"universal_login_experience,omitempty"`
+}
```

Comment:
Let's use the shorter `Prompt` for this struct to be more consistent with the codebase.